script.tag-generator/addon.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.tag-generator"
+       name="Tag Generator"
+       version="0.7.0"
+       provider-name="wellspokenman">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0"/>
+    <import addon="script.module.simplejson" version="3.3.0"/>
```

Comment:
Not needed anymore, please use only `json`

Reply:
done